public/cli/lib/commands/logger.ts

```diff
@@ -0,0 +1,83 @@
+import { join } from 'path';
+import fs from 'fs-extra';
+import { format as dateFormat } from 'date-fns';
+import winston from 'winston';
+import os from 'os';
+
+// Importing isDev causes an error running tests as we don't run in an
+// electron environment. Ava sets the NODE_ENV to 'test'.
+let isDev = false;
+if (process.env.NODE_ENV !== 'test') {
+  isDev = require('electron-is-dev');
```

Comment:
We shouldn't use `require` in TS. For dynamic imports you can do `isDev = (await import('electron-is-dev')).default;`, but then you would need to put it inside an `async` function and call it here because you can't `await` top-level code. This might be a good opportunity to move this to a utility file since I'm sure we will use this elsewhere but that could be done later when we have the need.